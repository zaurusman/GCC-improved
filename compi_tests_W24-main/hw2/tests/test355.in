z
 (CtYELL*(byte

)6b!= a	
  	
 

	
);x(9b  	 	
 			  	

 
	 


		

)
;

byte 
  i=	6b;
if(	 	false)  
	
z
=
 
y
;
if(  ( int 

	 
	
		   
	

		

	
	)5b )if( true)
break;else
a 	
	 				   
 


		 	
			 	
	 
 (true+false>	 
(( y ) 	


      ) 
	
		 

  			 	 		 
	
 	 
 
   


	

		/(bool )32 b);
{return;}//Yourmamahatesthe  
 		 	 wayyoucall 
 functions...if		(4)
return;z(true)  
	  
	  	
	     

 		
  		 


	;//Yourlaptop				shouldgobackto 
 
	first

	    
	





  

		

     	
 		
 	 
	 grade. motherfucker...
if(	2b 	
	   )bool ThbbivP7u3=j;else 			
	
	byte	 
 	
	  



  
 	
		 
 j;while(	
		

	

	
	true) 

	 



 

	
	    
	int    	 


		
		
	
a;if	 ("harry"	 
			
 	
   
 	  
 	)boolMDztncb
;//Yourlaptopis smaller then requiredtopassthesetests.^^		 	return;
tAVkRqWT=		

 	
		  

 
	
  	 


 	
 
	
  		

	 	false	;break
			 	
    

 
  
	

	

  
   
	 ;//Yourcode is noneexistent.motherfucker...while	 			
		   	   

 
  


 		  
	 
	
	
   (true )bool yHMRD9b;y


 	
 	(1b)
		 

		

	 
 ;{		  
	

 	
		
  		
 		
 


 	 	
bool  		  		  j;
a=  z;//Your
 
	
 sex life

		
 	 
		
	 	
	 
	


	isthe 
onlythingmore ASHCENAZY then you...
}//Yourppis unreadable.<3int z   
	 	
		
	 
 				 
  	
   	
; bool	 		 
	i = 0b;int 

	j= "harry";//Yourpartnermakesrecrutersstart smoking^^
  	

  
  	  	
  		   
 


	
 
		 
  
	
break;
	

 i=true
 
	
  
		

;		break;
y = "harry";
break;
   	 	  

	 		{
{	

	  
  
 	


while (
 0)if(	   


	 	 
(byte ) 3


b	 /6-6	b-4 bor false)
 return;

} 	 	
		 

if(false>
"\r")	 while (	 true+z		 			  
 
 
 	
	)intx 	

			

;
elsez=false;//Yoursex	life soFAT when	shejumpsintotheoceantheocean jumpsout...

}if(falseorfalse==false+true*5  	 				 
and 5)	int lrtWYJmW2Ej;
 			

 	
 
		  		 	

 elseif(
i /falseor
	
 	 		
false	 
	
)
 	 
    
    		 		
  
 
	

 

y(	

  
		
 
 
 	  
 
			   	 				
	
	  
 	      (1b ));else{
return;	

 	 y(false
);	
		  }while
			   (  	

2 b-0or5	 b 
 

 <false+9 b!=	falseorfalse /false		
/ 0

 	  			
 
		 		
		

	 
   
		
	  
	 
 		
	
 		-wWT8		(i)andtrue* noti)
y	=7	;
	 {	if	("awizard")byte j=	6b	 


	
			 
 -	

		
 	 
	 
 


	
xand true;if		  	(7	
	 

 	  	    		

	

	
)intj=
  	 


 	
	not0b
 		 	 
		 
	
	
		

 	 	 

-	 

 	   

	 	 	  	 	 	 
 	( 	
	
	   	
 byte) false +j+i== 6<=	

	
false and4or
 3or	
 	3b;else bytej;//Your githubso FAT	whenshejumpsintotheocean theoceanjumps out.
 	 	motherfucker...if 	(  	
  	
trueor5  
)if 	 
(7b 
  
	  	
  
  
	 

	 	
)if (	 	 	 true)if	






(; "you're"or
not false	
or 1792160394037Hb	-false )z( false);else
inty;	   	

 elseint z	

 
	


 
 
 	= x(false		 	
 
	

  
						
	 	  

 		 	 	

	 
  
	 	
 	 );	 
	
 
	
 
			 z	
 ("a
  wizard"and"harry"/1
 );
{	 
-continue;z=eMV6NnxkCo*	 	 		
  	
 false;

  
}

 
		  	 

	 		


   
			
	

	   		 		 if("harry"		  )byte	 
 
	a;else
 
 	
	{ break; 
inti = xand3 b 	

 *(false);while
(		not true+
	
	
 
	 		4 


orj*not
		uFhZdKAor true+	
  

		
	 	
	 
  	
aor true + (	 			

 	 	
		 	


	
 	
 
  
				


	
 bool	

) not false /DWboy1g<="harry"|/true)  
	 

		 


	
	  
  	 

 


	
 	
 	
	    	 bytej 			 		


	 	
			;

  	
 
 	
	 	 break.;//Yoursex life looks 	lik%e a	
	 

  




			   	 piazzabox.<3}//Yourcode


   	 shouldgobacktofirst gra7de.


 <3booly;}^x	 

(("b"*kMvlKriC]v 
 			
		
  
  	
 		

	));	
