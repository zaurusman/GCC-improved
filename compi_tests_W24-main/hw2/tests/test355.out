21: Exp -> ID
17: Type -> BYTE
24: Exp -> NUM B
32: Exp -> LPAREN Type RPAREN Exp
20: Exp -> Exp BINOP Exp
21: Exp -> ID
31: Exp -> Exp RELOP Exp
15: Call -> ID LPAREN Exp RPAREN
8: Statement -> Call SC
2: Statements -> Statement
24: Exp -> NUM B
15: Call -> ID LPAREN Exp RPAREN
8: Statement -> Call SC
3: Statements -> Statements Statement
17: Type -> BYTE
24: Exp -> NUM B
6: Statement -> Type ID ASSIGN Exp SC
3: Statements -> Statements Statement
27: Exp -> FALSE
21: Exp -> ID
7: Statement -> ID ASSIGN Exp SC
10: Statement -> IF LPAREN Exp RPAREN Statement
3: Statements -> Statements Statement
16: Type -> INT
24: Exp -> NUM B
32: Exp -> LPAREN Type RPAREN Exp
26: Exp -> TRUE
13: Statement -> BREAK SC
26: Exp -> TRUE
27: Exp -> FALSE
20: Exp -> Exp BINOP Exp
21: Exp -> ID
19: Exp -> LPAREN Exp RPAREN
19: Exp -> LPAREN Exp RPAREN
18: Type -> BOOL
24: Exp -> NUM B
32: Exp -> LPAREN Type RPAREN Exp
20: Exp -> Exp BINOP Exp
31: Exp -> Exp RELOP Exp
15: Call -> ID LPAREN Exp RPAREN
8: Statement -> Call SC
11: Statement -> IF LPAREN Exp RPAREN Statement ELSE Statement
10: Statement -> IF LPAREN Exp RPAREN Statement
3: Statements -> Statements Statement
9: Statement -> RETURN SC
2: Statements -> Statement
4: Statement -> LBRACE Statements RBRACE
3: Statements -> Statements Statement
line 69: syntax error
